Sources/SwiftGenKit/Parsers/StringsParser.swift

```diff
@@ -37,23 +37,58 @@ public final class StringsParser: Parser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parse(path: Path) throws {
-    let name = path.lastComponentWithoutExtension
+    if path.isDirectory {
 
-    guard tables[name] == nil else {
-      throw StringsParserError.duplicateTable(name: name)
-    }
-    guard let data = try? path.read() else {
-      throw StringsParserError.failureOnLoading(path: path.string)
-    }
+      if try hasMultipleLprojSubdirsWithDifferentLanguages(path: path) {
+        warningHandler?("Multiple .lproj directories found with different languages." +
+          "Duplicate keys will be parsed multiple times.", #file, #line)
+      }
+
+      try _ = path.children().map { childPath in
+        if childPath.isDirectory || isStringsFile(name: childPath.lastComponent) {
```

Comment:
`childPath` is already a `Path` (and not a String) here too. So instead of extract a string (`.lastComponent`) from that path only to later (line 90) construct an `NSURL(fileURLWithPath: name).pathExtension`, it's probably simpler to just pass `isStringsFile(path: childPath)`, and in the `func isStringsFile` implementation directly test `path.pathExtension == FileExtensions.strings`